src/app/components/navbar/navbar.component.html

```diff
@@ -0,0 +1,20 @@
+<div id="navbar" class="flex-container">
+    <div id="navbar-title" class="flex-container">
```

Comment:
Are there no styles linked to this css class? If not please remove it

Reply:
I have removed the class.